components/doc/calendar/floatlabeldoc.js

```diff
@@ -1,7 +1,7 @@
 import { DocSectionCode } from '@/components/doc/common/docsectioncode';
 import { DocSectionText } from '@/components/doc/common/docsectiontext';
 import { Calendar } from '@/components/lib/calendar/Calendar';
-import { FloatLabel } from '@/components/lib/floatlabel/FloatLabel';
+import { FloatLabel } from '@/components/lib/floatlabel/floatlabel';
```

Comment:
Are you sure this is supposed so be Lower case?  None of the other components are lower case?

Reply:
Good catch, This is addressed and fixed now